### PR TITLE
Amanda

### PR DESCRIPTION
**Pull request by AllenWang314**
Major changes to FaqPage.tsx and server/controllers/user.py to limit leaderboard to top 10 sorted

### CODE REVIEW

File: client/src/components/FAQPage.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        'Please reach out in the slack to hackers or mentors. If you are still having difficulties please reach out to a team member at blueprint@hackmit.org.'
+    ]
+  },{
+    key: 'q6',
+    title: {
+      content: 'What if I have more questions or need help using helpq? ',
+      icon: 'star',
+    },
+    content: [
+        'Reach out to us in the slack or send us an email at blueprint@hackmit.org!'
```
Comment (TheReddKing) [inline client/src/components/FAQPage.tsx:59]: switch email to `settings.app_contact_email` where settings is
` const { settings } = useViewer();`
This way the FAQ page is modular

File: client/src/components/FAQPage.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    ]
+  },
+]
+
+const FAQPage = () => {
```
Comment (TheReddKing) [inline client/src/components/FAQPage.tsx:64]: `const { settings } = useViewer();` here, which means panels might have to be be in the class component.

File: server/controllers/users.py [hunk 3]
```diff
@@ -32,11 +33,13 @@ def mentor_rankings():
         unrated_tickets = Ticket.query.filter_by(claimant=user, status=3).all()
         totalUnrated += len(unrated_tickets)
 
-        if (totalRatings > 0):
-            ret.append({"name": user.name, "rating": "{:.1f}".format(
+        if (totalRatings > 0): # "{:.1f}".format
```
Comment (TheReddKing) [inline server/controllers/users.py:36]: are u not doing precision anymore? Would be careful about that.

**Review by TheReddKing — CHANGES_REQUESTED** [review]
Just removing the email and replacing it with the variable should be good!

File: server/controllers/users.py (hunk 3 above)
Comment (AllenWang314) [reply server/controllers/users.py:36]: was moved to frontend in latest commit, moved back here in revisions. had to keep as float since we were sorting

**Review by TheReddKing — APPROVED** [review]

**Review by TheReddKing — APPROVED** [review]

**Pull request merged**